podcasts/AppDelegate.swift

```diff
@@ -35,6 +36,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         configureFirebase()
         TraceManager.shared.setup(handler: traceHandler)
 
+        // Facebook AEM Analytics
+        FacebookCore.Settings.shared.appID = ApiCredentials.instagramAppID
+        ApplicationDelegate.shared.application(application, didFinishLaunchingWithOptions: launchOptions)
```

Comment:
can we move this before the `return true` and put it behind `FeatureFlag.podcastNewformAppsFlyer.enabled`?